### DISCUSSION TITLE
Tracking down slowness

### DISCUSSION THREAD

**Discussion by dkegel-fastly** [golangci/golangci-lint#2126] [Q&A] [answered]
golangci-lint seems to be slow (on first run, and thus in our ci?) on a small codebase (that imports a large one, probably)

--timeout 8m wasn't enough, I had to double it to --timeout 16m

This is with a fair number of linters enabled.

What are the steps for tracking down slowness?  
(e.g. how does one clear the build cache, and display how long each linter takes?)

**Comment by ldez** [comment]
Hello,

you can use the verbose mode:
```bash
golangci-lint run -v
```

**Reply by Antonboom** [reply]
@ldez, the verbose flag does not show statistics **per linter**.
Maybe add this? Quite often requested from people.

**Reply by ldez** [reply, marked as answer]
1. it's more complex than just counting by linter because there are conditional shared steps
2. the debug flags can be used to have similar details https://golangci-lint.run/contributing/debug/
